### ISSUE TITLE
`manage.py migrate` fails due to conflicting migrations

### ISSUE THREAD

**Issue by jwodder** [dandi/dandi-publish#143]
Currently (latest commit 1296524), running `docker-compose run --rm django ./manage.py migrate` in this repository fails with:

```
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0009_remove_description, 0010_timestampedmodel in publish).
To fix them run 'python manage.py makemigrations --merge'
```

**Comment by yarikoptic** [comment]
Confirming seeing it on the same commit.  The one which worked for me was 06078de .

**Comment by brianhelba** [comment]
Fixed by #144.
